packages/pact-testing/src/helpers/shell-helpers.sh

```
@@ -0,0 +1,225 @@
+
+log_warning() {
+    echo -e "${YELLOW}⚠️  $1${NC}"
+}
+
+log_error() {
+    echo -e "${RED}❌ $1${NC}"
+}
+
+# Create timestamped results directory
+create_test_results_dir() {
+    local TIMESTAMP=$(date +%Y%m%d_%H%M%S)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)